.travis.yml

```diff
@@ -47,6 +47,9 @@ matrix:
 
 install:
   - mvn --version
+  - cd kyuubi-shaded
+  - mvn clean install
+  - cd -
```

Comment:
Can it be replaced by `mvn clean install -pl <modules> -am -Pkyuubi-shaded`?

Reply:
`mvn clean install -pl :kyuubi-shaded-curator,:kyuubi-shaded-hive-service-rpc -am -Pkyuubi-shaded`